src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html

```diff
@@ -0,0 +1,22 @@
+<div #chartElement id="{{chartId}}" class="deployments-donut-chart" [ngClass]="{'mini': mini}"></div>
+<div *ngIf="mini" class="deployments-donut-chart-mini-text">
+  <span *ngIf="!idled">
+    {{total}}
```

Comment:
Can you use `*ngIf="!idled; else idled;"` and `#idled` here?

https://angular.io/api/common/NgIf